sourcerer-core/src/main/java/org/elder/sourcerer2/subscription/SessionSubscriber.java

```diff
@@ -31,7 +31,7 @@ public void onNext(final EventSubscriptionUpdate<T> update) {
         try {
             switch (update.getUpdateType()) {
                 case EVENT:
-                    logger.debug("Offering event {}", update.getEvent());
+                    // logger.debug("Offering event {}", update.getEvent());
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.elder.org/coding_rules#rule_key=squid%3ACommentedOutCodeLine)